src/Frame/DataBuilderFrame.py

```diff
@@ -106,36 +117,118 @@ def GetPathToFiles(self):
     def debugPrint(self, evt=None):
         print('Debug: ' + self.pathToFiles)
 
-
-class DisplayData(tk.Frame):
+class ChooseColumns(tk.Frame):
     def __init__(self, root, pathToCSV):
-        """Initialize a Data Displayer for displaying the data from a specific CSV"""
         if not os.path.isfile(pathToCSV):
-            raise Exception('Path to CSV: ' + pathToCSV + ' does not exist or is not a directory.')
+            raise Exception('Path to CSV: ' + pathToCSV + ' does not exist or is not a file.')
 
-        super().__init__(root, highlightbackground="blue", highlightthickness=2)
+        super().__init__(root, highlightbackground="pink", highlightthickness=2)
         self.root = root
         self.pathToCSV = pathToCSV
         self.df = None
-        self.readData()
+        self.unchosenColumns = []
+        self.chosenColumns = []
+        self.getAllCols()
+        self.unchoseColVar = tk.StringVar(value=self.unchosenColumns)
+        self.choseColVar = tk.StringVar(value=self.chosenColumns)
+
+        self.unchoseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.unchoseColVar)
+        self.unchoseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+        self.choseColListbox = tk.Listbox(self, selectmode=tk.SINGLE, listvariable=self.choseColVar)
+        self.choseColListbox.pack(side=tk.LEFT, expand=tk.NO, fill=tk.BOTH)
+
+        self.addColButton = tk.Button(self, text="Add Column", command=self.addCol)
+        self.addColButton.pack()
+        self.removeColButton = tk.Button(self, text="Remove Column", command=self.removeCol)
+        self.removeColButton.pack()
+
+    def getAllCols(self):
+        self.df = pd.read_csv(self.pathToCSV)
```

Comment:
```suggestion
        self.df = pd.read_csv(self.pathToCSV, nrows=2)
```
This will read in the entire file otherwise. Don't want to do that.